.storybook/preview.js

```diff
@@ -1,5 +1,6 @@
 import SpiritTheme from './spirit.theme';
 import '../.storybook/assets/stylesheets/index.scss';
+import { IconGlobalDecorator } from './IconGlobalDecorator';
```

Comment:
Great. Just I will suggest moving this file to `.storybook/decorators` directory.

Reply:
done

Reply:
Are you certain? I see no change in this particular code. Please, check if your modifications are pushed first and then use the "Resolve" button.

Reply:
Those comments are more like a checklist for me, that I have it done, then when I process everything, I will give another review request.